pkg/daemon/ceph/client/upgrade_test.go

```diff
@@ -69,12 +82,90 @@ func TestEnableNautilusOSD(t *testing.T) {
 	executor.MockExecuteCommandWithOutput = func(debug bool, name string, command string, args ...string) (string, error) {
 		assert.Equal(t, "osd", args[0])
 		assert.Equal(t, "require-osd-release", args[1])
-		assert.Equal(t, "nautilus", args[2])
-		assert.Equal(t, 3, len(args))
 		return "", nil
 	}
 	context := &clusterd.Context{Executor: executor}
 
 	err := EnableNautilusOSD(context)
 	assert.Nil(t, err)
 }
+
+func TestOkToStopDaemon(t *testing.T) {
```

Comment:
This needs a negative/error condition test.

Reply:
There is no real negative condition, if it's negative then it's skipped.

Reply:
There is no test to make sure this function returns error when it is supposed to.

Reply:
The only that can happen is the under-lying function that fails, I'm not sure how to mock that.